client/python/openlineage/client/facet.py

```diff
@@ -420,3 +420,51 @@ class ExtractionErrorRunFacet(BaseFacet):
     @staticmethod
     def _get_schema() -> str:
         return SCHEMA_URI + "#/definitions/ExtractionErrorRunFacet"
+
+
+@attr.define
+class Tag(RedactMixin):
```

Comment:
We shouldn't add new code here - `facets.py` is for backwards compatibility, new code should use autogenerated facets.